Source/WebCore/platform/network/cocoa/ResourceRequestCocoa.mm

```diff
@@ -64,7 +64,8 @@
 ResourceRequest::ResourceRequest(ResourceRequestPlatformData&& platformData, const String& cachePartition, bool hiddenFromInspector)
 {
     if (platformData.m_urlRequest) {
-        setRequester(*platformData.m_requester);
+        if (platformData.m_requester)
+            setRequester(*platformData.m_requester);
```

Comment:
Should we provide a default value if `platformData.m_requester` is not set, like this?

```
        setRequester(*platformData.m_requester.value_or(ResourceRequestRequester::Unspecified);
```

Reply:
That's a good point. But I think that's already happening as the default value would be the first value in the enum(ResourceRequestRequester::Unspecified)

Reply:
WebCore::ResourceRequestRequester::Unspecified is already the default value, so that seems unnecessary.